sdk/OoyalaSkinSDK/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -274,6 +275,16 @@ - (void) bridgeAdStartNotification:(NSNotification *)notification {
   }
 }
 
+- (void) bridgeAdTappedNotification:(NSNotification *)notification {
```

Comment:
Can you add a comment in OOSkinViewController for the bridgeAdTappedNotification, to communicate to others when that should be expected (Only IMA)